app/controllers/agent_statuses_controller.rb

```diff
@@ -2,7 +2,15 @@
 class AgentStatusesController < ApplicationController
   def index
     @agent_statuses = AgentStatus.where(open: true).joins(agent: :team)
-    @agent_statuses.each { |a| a.status = normalize_status(a.status) }
+    lunched = Rails.cache.fetch('lunched', force: true)
```

Comment:
Create empty array here to simplify code later on. Also consider using the Ruby's `Set` object.